src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -206,6 +207,13 @@ private void initializeVcfWriter() {
         final Set<VCFHeaderLine> hInfo = new LinkedHashSet<>();
         hInfo.addAll(getHeaderForVariants().getMetaDataInInputOrder());
 
+        // need AC, AN and AF since output if set filtered genotypes to no-call
+        if ( setFilteredGenotypesToNocall ) {
```

Comment:
I assume you mean the same between `SelectVariants` and `VariantFiltration`. I added the method `void addChromosomeCounts(final Set<VCFHeaderLine> headerLines)` to `GATKVariantContextUtils`.